src/test/java/org/tests/query/other/TestQuerySingleAttribute.java

```diff
@@ -62,11 +62,10 @@ public void withOrderBy() {
         .setDistinct(true)
         .select("name")
         .where().eq("status", Customer.Status.NEW)
-        .orderBy().asc("name")
-        .setMaxRows(100);
+        .orderBy().asc("name");
```

Comment:
sqlserver produces "select top 100..." instead of "select ... limit". So I removed this, so that the test will easier pass the sqlserver platform